clang/lib/Driver/ToolChain.cpp

```diff
@@ -196,6 +196,16 @@ bool ToolChain::defaultToIEEELongDouble() const {
   return PPC_LINUX_DEFAULT_IEEELONGDOUBLE && getTriple().isOSLinux();
 }
 
+static void
+processARMAArch64MultilibCustomFlags(Multilib::flags_list &List,
```

Comment:
Is this system intended to be permanently specific to Arm and AArch64? I don't really see why it should be: the need to build libraries distinguished by a configuration option that isn't a standard compile flag is certainly not specific to Arm. Perhaps this should just be called `processMultilibCustomFlags`?

Reply:
Fixed